circuit/js/src/scaffold.ts

```diff
@@ -81,19 +81,16 @@ export abstract class AxiomBaseCircuitScaffold<T> extends BaseCircuitScaffold {
     const vk = convertToBytes32(partialVk);
     const packed = encodePacked(
       ["uint8", "uint16", "uint8", "bytes32[]"],
-      [this.config.k, this.numInstances / 2, vk.length, vk as `0x${string}`[]],
+      [this.config.k, this.resultLen, vk.length, vk as `0x${string}`[]],
     );
     const schema = keccak256(packed);
     return schema as string;
   }
 
   prependCircuitMetadata(config: CircuitConfig, partialVk: string[]): string[] {
-    if (config.numInstance !== 1) {
-      throw new Error("`numInstance` only supported value is 1");
-    }
     const encodedCircuitMetadata = encodeAxiomV2CircuitMetadata({
       version: 0,
-      numInstance: [config.numInstance],
+      numValuesPerInstanceColumn: [2 * this.resultLen + 16 * AxiomV2CircuitConstant.UserMaxSubqueries],
```

Comment:
this should be `2 * AxiomV2CircuitConstant.UserMaxOutputs + 16 * AxiomV2CircuitConstant.UserMaxSubqueries`: it does not depend on the `resultLen`.

your formula in the encoder test is correct: ` AxiomV2CircuitConstant.UserMaxOutputs * AxiomV2CircuitConstant.UserResultFieldElements + 
        AxiomV2CircuitConstant.UserMaxSubqueries * SUBQUERY_RESULT_LEN`

Reply:
sorry I realize this is because I wrote a typo in my previous comment.